articles/055_ros_parameter_design.md

```diff
@@ -109,17 +109,28 @@ Each parameter consists of a key and a value.
 The key is a string value.
 The value can be one of the following datatypes:
 
-- `float64`
+- `bool`
 - `int64`
+- `float64`
 - `string`
-- `bool`
-- `bytes[]`
+- `byte[]`
+- `bool[]`
+- `int64[]`
+- `float64[]`
+- `string[]`
 
-The datatypes are chosen as non-complex datatypes, as defined in the [interface definitions article](articles/interface_definition.html)
+The datatypes are chosen as non-complex datatypes, as defined in the [interface definitions article](interface_definition.html)
```

Comment:
This link was going to `articles/articles/interface_definition.html`.

Reply:
nitpick: A `.` (dot) is missing here.

Reply:
As in, referring to the current directory? Will add.

Reply:
Or did you mean a full stop/period at the end of the sentence?

Reply:
yes, I meant punctuation.